sdk/src/ingredient.rs

```diff
@@ -1563,12 +1564,13 @@ mod tests {
         assert_eq!(&ingredient.title, title);
         assert_eq!(ingredient.format(), format);
         assert!(ingredient.manifest_data().is_some());
-        assert!(ingredient.metadata().is_none());
-        assert!(ingredient.validation_status().is_none());
+        assert_eq!(ingredient.metadata(), None);
```

Comment:
These fixes are so weird.  In a prior Rust release it had us change the tests the other way around